ruby/CMakeLists.txt

```diff
@@ -116,19 +87,34 @@ target_link_libraries(openstudio_rb
 )
 
 if( WIN32 )
-  target_link_libraries(openstudio_rb ${RUBY_MINGW_STUB_LIB})
+  target_link_libraries(openstudio_rb PRIVATE ${RUBY_MINGW_STUB_LIB})
 endif()
 
 if (APPLE)
   target_link_libraries(openstudio_rb
+    PRIVATE
     ${OPENSSL_CRYPTO_LIBRARY}
     ${OPENSSL_SSL_LIBRARY}
   )
 endif()
 
-install(TARGETS openstudio_rb DESTINATION Ruby COMPONENT "RubyAPI")
+#export(TARGETS
+#  openstudio_rb
+#  NAMESPACE "openstudio::"
+#  FILE "${PROJECT_BINARY_DIR}/openstudioConfig.cmake"
+#)
```

Comment:
Remove?

Reply:
ah yes. I hate random commented code as much as the next person. https://github.com/NREL/OpenStudio/commit/676874f7e1a13cda2ffdcb697a1d461cc23fddaa